### PR TITLE
Disable theming by default on Linux/GTK

### LINKED ISSUE #629: Disable theming by default in Linux distributions?
Currently, on Linux, most contributors seem to either use dark theme or use no theme. There seems to be a relative consensus that the light theme on Linux is not better than no theme. So I would recommend that -on Linux only- we just disable theming by default.
Thoughts?

### PR DESCRIPTION
**Pull request by mickaelistria**
The light theme shows no clear value over no theme on Linux, so let's switch to no theme by default on Linux.

### CODE REVIEW

**Comment by mickaelistria** [comment]
@iloveeclipse @akurtakov should we merge this milestone?

**Comment by akurtakov** [comment]
I don't mind it personally.

File: bundles/org.eclipse.e4.ui.workbench.swt/src/org/eclipse/e4/ui/workbench/swt/internal/ThemePreferenceInitializer.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import org.eclipse.core.runtime.preferences.DefaultScope;
+import org.eclipse.core.runtime.preferences.IEclipsePreferences;
+import org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine;
+import org.eclipse.e4.ui.internal.workbench.swt.WorkbenchSWTActivator;
+
+public class ThemePreferenceInitializer extends AbstractPreferenceInitializer {
+
+	@Override
+	public void initializeDefaultPreferences() {
+		IEclipsePreferences preferences = DefaultScope.INSTANCE
```
Comment (iloveeclipse) [inline bundles/org.eclipse.e4.ui.workbench.swt/src/org/eclipse/e4/ui/workbench/swt/internal/ThemePreferenceInitializer.java]: Please test this with -pluginCustomization /path/to/pluginCustomization.ini and check if the product settings will or will not be considered here.

Product defaults from ini should take precedence.

**Comment by mickaelistria** [comment]
I've updated the patch to properly try it (my previous tests were actually faulty and giving a false impression it worked...); and then with this newer patch, I can verify that the settings from plugin_customization.ini take higher priority.

**Comment by mickaelistria** [comment]
For the moment, I'm a bit discouraged by the test failures and test assuming that the IDE is started with theme enabled. I don't plan to work on this in short-term. So if anyone is interested in continuing it before I come back to it (can't tell when), then thank you, be my guest!

**Comment by akurtakov** [comment]
As @mickaelistria doesn't plan to continue on this one I'm closing it.

**Pull request closed without merging**